### PR TITLE
(#444) Take out Read-Host in Client Setup

### LINKED ISSUE #444: Make Client-Setup Script for C4B Azure More Automation Friendly
## What New Or Updated Would You Like To See?

Would like to update the Client-Setup script for our C4B Azure Enviornment  so that it mirrors the other client-setup files in that they do not prompt for input, but rather encourage you to set the values directly in the script. 

## Why Is It Needed?

While working support tickets a few customers have asked how to edit the script to make it easier to put straight into an RMM/Deployment mechanism they have to deploy the setup script to their endpoint machines.

### PR DESCRIPTION
**Pull request by ryanrichter94**
## Description Of Changes
Remove Read-Host from Client Setup script and correct handling of pscredential object as a result.

## Motivation and Context
Make this Client Setup script more closely match the format used by QSG and make it easier to define one and plop into an RRM/Code Deployment tool to run on endpoint machines.

## Testing

* [X] I have previewed these changes using the [Docker Container](https://github.com/chocolatey/docs/tree/master/.devcontainer) or another method before submitting this pull request.

## Change Types Made

* [ ] Minor documentation fix (typos etc.).
* [X] Major documentation change (refactoring, reformatting or adding documentation to existing page).
* [ ] New documentation page added.

## Change Checklist

* [ ] Requires a change to menu structure (top or left hand side)/
* [ ] Menu structure has been updated

## Related Issue

Fixes #444

### CODE REVIEW

File: input/en-us/quick-deployment/azure/client-setup.md [hunk 1]
```diff
@@ -71,15 +71,15 @@ When you're ready, run the following on the client from an elevated (Run as Admi
...
 ```powershell
 # Prompt for input
-$fqdn = Read-Host 'Enter the FQDN for your Chocolatey for Business QDE Azure Environment'
-$clientCommunicationSalt = Read-Host 'Enter the "ccmClientCommunicationSalt", you can find this in your Azure Key Vault'
-$serverCommunicationSalt = Read-Host 'Enter the "ccmServiceCommunicationSalt", you can find this in your Azure Key Vault'
-$password = Read-Host 'Enter the "ChocoUserPassword", you can find this in your Azure Key Vault' -AsSecureString
+$fqdn = 'Enter the FQDN for your Chocolatey for Business QDE Azure Environment'
+$clientCommunicationSalt = 'Enter the "ccmClientCommunicationSalt", you can find this in your Azure Key Vault'
+$serverCommunicationSalt = 'Enter the "ccmServiceCommunicationSalt", you can find this in your Azure Key Vault'
+$password = 'Enter the "ChocoUserPassword", you can find this in your Azure Key Vault'
```
Comment (JPRuskin) [inline input/en-us/quick-deployment/azure/client-setup.md:77]: It's worth noting that removing `Read-Host` in these cases will result in the variable being set to this string, and not prompting the user for input. 

It's possible we would want to replace the strings with more obvious `<REPLACE-ME-WITH-YOUR-FQDN>` type stuff (or even just trimming it down to the `ccmClientCommunicationSalt` etc) if we wanted to do this.

Comment (ryanrichter94) [reply input/en-us/quick-deployment/azure/client-setup.md]: fixed

**Comment by Windos** [comment]
@ryanrichter94, looks like this'll need a rebase but other than that looks good (I'll pop my approval on it after the branch is updated otherwise it'll be reset anyway 😅)

**Review by gep13 — APPROVED** [review]
LGTM!

**Comment by GhostPirateBob** [comment]
@gep13 Just a guy browsing the docs and noticed if you try to run this you'll get:

```ConvertTo-SecureString: A parameter cannot be found that matches parameter name 'AsPlainTest'.```

Probably want ```-AsPlainText``` ;)

**Comment by gep13** [comment]
> @GhostPirateBob said...
Probably want -AsPlainText ;)

Doh! Well, that is a little embarrassing! I have just made a change to get this updated, and it should be live on the site soon.  Thanks for letting me know!

**Comment by gep13** [comment]
@GhostPirateBob this change is now live on the site.

**Pull request merged**